From 4b3a8f347179d7e2a1062eeed13a6474deb28728 Mon Sep 17 00:00:00 2001
From: Chen Qi <Qi.Chen@windriver.com>
Date: Thu, 16 Oct 2025 08:48:49 +0800
Subject: [PATCH] Add clock group

The clock group is needed to "to enable applications like linuxptp
to open clocks without root privileges". See the link below:
https://github.com/systemd/systemd/commit/af96ccfc24bc4803078a46b4ef2cdeb5decdfbcd

Note that this group is needed as long as people use udev from
systemd because 50-udev-default.rules needs it. See the above
commit.

Upstream-Status: Denied [https://salsa.debian.org/debian/base-passwd/-/merge_requests/17]

Signed-off-by: Chen Qi <Qi.Chen@windriver.com>
---
 group.master | 1 +
 1 file changed, 1 insertion(+)

diff --git a/group.master b/group.master
index e54fd1d..6f84539 100644
--- a/group.master
+++ b/group.master
@@ -39,5 +39,6 @@ staff:*:50:
 games:*:60:
 shutdown:*:70:
 wheel:*:80:
+clock:*:81:
 users:*:100:
 nogroup:*:65534:
-- 
2.34.1

